examples/echo/main.c

```diff
@@ -143,16 +144,35 @@ static void handle_rx (YggWorker   *worker,
   g_bytes_unref (data);
 }
 
+static gboolean
+sigint_callback(gpointer main_loop) {
```

Comment:
I noticed this should have a space following the function name.
```suggestion
sigint_callback (gpointer main_loop) {
```